### ISSUE TITLE
Integrate Zenodo community into the UI

### ISSUE THREAD

**Issue by nuest** [o2r-project/o2r-platform#180]
## Display

Show in compendium listings and detail pages if a compendium is part of the [Zenodo community "o2r"](https://sandbox.zenodo.org/communities/o2r/), ideally with direct links to (a) the Zenodo repository, as well as (b) the community landing page.

## Publication workflow

- Create a list of shipped compendia visible only for Admins
- If a compendium is shipped but not part of the community yet, add an "Accept new asset" button to admin page
- Clicking the button starts the acceptance workflow in Zenodo

Not sure what server-side code this requires, but it probably should go into shipper, and should be driven by UI requirements, so I suggest to outline the desired user workflow first.

@7048730 ideas, comments? Can you create an issue with shipper to document what kind of authentication the shipper must store and which API calls it must make?

**Comment by 7048730** [comment]
With zenodos current API it is possible to automatically add a new record to a community (via presence of the corresponding metadata), but as owner of this community you cannot confirm new entries via API. This is done with notification emails and web interface buttons.

[developers.zenodo.org](http://developers.zenodo.org/#representation)

`communities`
_array of objects_ 
_(optional)_

```
List of communities you wish the deposition to appear. 
The owner of the community will be notified, and can either accept or reject your request. 
```
Each array element is an object with the attributes:
_identifier_: Community identifier

**Comment by 7048730** [comment]
since the feature is already there in zenodo, but was not documentated (or I thoroughly overlooked it),
the shipper can take this task
https://github.com/o2r-project/o2r-shipper/issues/43

**Comment by 7048730** [comment]
but first we need to decide, what gets accepted and what gets rejected, e.g. only erc

**Comment by nuest** [comment]
I agree, only ERCs should be accepted... so an examination using the Zenodo DOI should be done before accepting it.
